Cargo.toml

```diff
@@ -166,8 +166,8 @@ tempfile = "3.8.1"
 
 # Published deps
 #
-# https://github.com/gear-tech/dlmalloc-rust/tree/v0.1.4
-dlmalloc = { package = "gear-dlmalloc", version = "0.1.4" }
+# https://github.com/gear-tech/dlmalloc-rust/tree/v0.1.5
+dlmalloc = { package = "gear-dlmalloc", version = "0.1.5" }
```

Comment:
Bump version to 0.2.0 due to breaking changes (see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)

Reply:
Done